templates/inc/head.tpl

```diff
@@ -1,3 +1,9 @@
+{locale path="Translations" domain="AllTube"}
+<?php
+$language = "zh_CN";
+putenv("LANG=".$language);
+setlocale(LC_ALL, $language);
+?>
```

Comment:
PHP code in template files won't be executed.
You need to put this in index.php for example.